### ISSUE TITLE
[Design] - Profile: direct deposit documentation improvements

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/va.gov-team#41528]
In #40862 we audited the direct deposit documentation and identified improvements.  We'll tackle as much as we can in this sprint and break out what doesn't get completed for the next.

#### Product Outline

- [x]  Update to include latest updates to conditional logic

#### All docs

- [ ] Clarify abbreviations and add links where it makes sense (e.g. spelling out the full name of a backend service and linking to docs about it)

#### Product Outline

- [x] Add a "What to know about the product" section or similar in Feature Overview section; something easily scannable/readable so anyone (esp. people not on our team) can understand what it does.
   - [This is a good example of info](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/direct-deposit/what-benefits-use-which-direct-deposit.md) I think should be in this type of section
- [x] Any legacy info about the product that might be helpful
- [x]  Create a "getting started" BE engineering doc (add a placeholder for now)

#### Use cases
- [ ] Document all use cases (not just edge cases)
#### Flow documentation
- [x] Wireflow of feature for visual overview

#### Misc
- [ ] Create reference to direct deposit documentation from profile product folder

**Comment by andaleliz** [comment]
@SKasimow I went ahead and created this ticket for sprint 74, FYI

**Comment by andaleliz** [comment]
Hi @Samara-Strauss, I'm working on the docs, and it seems that users can't _remove_  or turn off direct deposit from profile. Is that correct? If so, how do people go about doing that?

**Comment by andaleliz** [comment]
Hi @Samara-Strauss, I'm working on the docs, and it seems that users can't _remove_  or turn off direct deposit from profile. Is that correct? If so, how do people go about doing that?

Also, what would be the easiest way for you to review my updates to the product outline? I'm thinking I can commit them on a separate branch and then you can compare changes if needed. I could also put them in a google doc so we can use commenting if needed. Happy to do whatever works!

**Comment by Samara-Strauss** [comment]
> I'm working on the docs, and it seems that users can't remove or turn off direct deposit from profile. Is that correct? If so, how do people go about doing that?

Correct, there is not a way to turn off direct deposit from the profile. I think there is a way for people to go from direct deposit to paper check, but it's discouraged. I believe the government has a mandate to try to get as many people using direct deposit as possible, so while we used to have info on the content page for direct deposit about how to switch back to paper checks, that's since been taken down. I don't know for sure whether that form exists still or not, but my guess is that it does.

> Also, what would be the easiest way for you to review my updates to the product outline? 

Google doc might be best so we can iterate. I'm not savvy enough to do that via branches :)

**Comment by andaleliz** [comment]
@Samara-Strauss I shared the google doc with you via email, and left comments about what I've added. Note that the formatting is a little wonky, as the markdown > google doc transition isn't super smooth. I'll make sure all the heading levels are correct + formatting matches what we have now when we move it back into google docs.  I look forward to your feedback!

**Comment by Samara-Strauss** [comment]
Great! I'll check it out. And no worries about the formatting. I totally get that things don't play nice transitioning from Github to Google Docs.

**Comment by Samara-Strauss** [comment]
This looks great! I made a few comments and suggestions, but I think this is great.

**Comment by Samara-Strauss** [comment]
Great!! Thank you so much!

> I added a note to the [fraud error documentation](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/direct-deposit/fraud-handling/routing-number-flagged-for-fraud-error.md) but do we even need to keep it around?

The TL;DR answer to your question is yes, that page is still sort of relevant. But this is the longer explanation:

So, there are two codes in that doc. One is `cnp.payment.routing.number.fraud.message`, which is now obsolete. This error showed up if someone entered a routing number that was part of a fraud ring in 2015. While the numbers were legit, because they'd been part of this fraud ring, there was an extra flag on people trying to use them. Since this flag has been removed (the fraud team has other means of protecting people's information now), the code is no longer relevant to us.

However, the second code -- `cnp.payment.flashes.on.record.message`  -- is still relevant. It applies to *individual accounts* that are flagged for fraud, so when that user tries to save direct deposit information, they'll get an error saying they can't. This is a use case under which this would happen:

- A person suspects that their direct deposit information has been compromised.
- They call the VA and report this. The fraud team puts a flag on that person's information which prevents them from updating their information online.
- Once the issue is resolved, the flag is removed for that person.

Let me know what questions you have about that!

**Comment by andaleliz** [comment]
I've created the wireflow in our direct deposit Sketch files.  There are some OS/Sketch version challenges ([Slack post](https://dsva.slack.com/archives/C01KY4VUN8P/p1653671518836789)) that I want to make sure Christina doesn't struggle with while I'm out, so I worked on this locally and **have not pushed this to the Sketch for Teams cloud yet.**. I've attached a PDF of the user flow for review, @Samara-Strauss and @SKasimow. 
[https://app.zenhub.com/files/133843125/199434f8-6eb1-4b2d-8a4e-6b617065061a/download](https://app.zenhub.com/files/133843125/199434f8-6eb1-4b2d-8a4e-6b617065061a/download)
Once I'm sure Christina is up and running w/ the latest OS and Sketch version, I'll merge this into our existing DD file which will be viewable in a browser via a link.
